demos/browser/app/meetingV2/meetingV2.ts

```diff
@@ -1445,9 +1470,43 @@ export class DemoMeetingApp implements
   async populateAllDeviceLists(): Promise<void> {
     await this.populateAudioInputList();
     await this.populateVideoInputList();
+    await this.populateVideoFilterInputList();
     await this.populateAudioOutputList();
   }
 
+
+  private async populateVideoFilterInputList(): Promise<void> {
+    const genericName = 'Filter';
+    let filters: VideoFilterName[] = ['None'];
+
+    if (this.defaultBrowserBehaviour.supportsCanvasCapturedStreamPlayback() && this.enableUnifiedPlanForChromiumBasedBrowsers) {
+      filters = filters.concat(VIDEO_FILTERS);
+      if (this.defaultBrowserBehaviour.supportsVideoProcessingBasedOnTensorflowModel()) {
+        filters.push('Segmentation');
+      }
+    }
+
+    this.populateInMeetingDeviceList(
+      'dropdown-menu-filter',
+      genericName,
+      [],
+      filters,
+      undefined,
+      async (name: VideoFilterName) => {
+        this.selectedVideoFilterItem = name;
+        this.log(`clicking video filter ${this.selectedVideoFilterItem}`);
+        this.toggleButton('button-video-filter', this.selectedVideoFilterItem === 'None' ? 'off' : 'on');
+        if (this.isButtonOn('button-camera')) {
+          try {
+            await this.openVideoInputFromSelection(this.selectedVideoInput, false);
+          } catch (err) {
+            this.log('Failed to choose VideoTransformDevice', err);
```

Comment:
Should you call `fatal` here? We don't expect any of them to fail, right?

Reply:
added fatal